react-app/src/components/UserProfile.jsx

```diff
@@ -0,0 +1,58 @@
+import React, { useState } from "react";
+import { Container, Grid, Paper, Typography } from "@mui/material";
+import axiosInstance from "../axios";
+import { base } from "../api";
+import { useEffect } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import { signIn } from "../store/auth";
+
+const UserProfile = () => {
+  const [admin, setAdmin] = useState([]);
+  const dispatch = useDispatch();
+  const auth = useSelector((state) => state.auth);
+
+  const getUser = async () => {
+    try {
+      const { data } = await axiosInstance({
+        method: "get",
+        url: base + "/auth/get-user-info",
+      });
+      dispatch(signIn(data.data.attributes));
+      setAdmin(data);
+      console.log(admin?.data?.attributes);
+    } catch (errro) {
+      console.log("not successful");
+    }
+  };
+
+  const userInfo = auth?.user;
```

Comment:
There's no need to declare another variable to access `user` in your redux state. When you use `useSelector` you can use object destructuring, like so:
```js
const { user } = useSelector((state) => state.auth);
```

Reply:
done!